tests/Fixtures.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
```

Comment:
We generally use define include guards instead of `#pragma once`

Reply:
I'm open to using `#pragma once` but we should be consistent through out the library, all of the other headers define `FILENAME_H`

Reply:
I'm more biased to pragma once, but I think there enough changes here for now.